tap_stripe/__init__.py

```diff
@@ -428,6 +437,16 @@ def get_bookmark_for_stream(stream_name, replication_key):
             int(utils.strptime_to_utc(Context.config["start_date"]).timestamp())
     return stream_bookmark
 
+def get_bookmark_for_sub_stream(stream_name):
+    sub_stream_name = stream_name
+    stream_name = PARENT_STREAM_MAP[stream_name]
```

Comment:
if it is a parent stream name then change the variable name.

Reply:
Updated the code